aries-site/src/pages/learn/grid-fundamentals-part-one.mdx

```diff
@@ -0,0 +1,198 @@
+import { Notification, Paragraph } from 'grommet';
+import {
+  StepOne,
+  StepTwo,
+  StepThree,
+  StepFour,
+} from '../../examples/learn/tutorials/grid-fundamentals-part-1';
+
+In this tutorial you will use Grid to display a collection of products, similar to
+that of an e-commerce website, and use its [properties](https://v2.grommet.io/grid?theme=hpe#props)
+to:
+
+- Present a collection of items in a grid.
+- Allow the number of grid columns to adapt to a variety of screen widths.
+- Allow the items to adapt and flex within the available space.
+
+At the end of this tutorial, you will have built this:
+
+<!-- show example of product cards in a grid -->
+
+## Prerequisites
+
+This tutorial can be completed in its entirety by following the steps below. However,
+because Grommet is a ReactJS-based UI library, introductory knowledge of
+[React](https://reactjs.org/docs/getting-started.html) may be helpful in understanding
+the syntax.
+
+## Getting started
+
+Let’s get started. Use this [CodeSandbox template]() as a starter.
+
+First, in `App.js` add the Grid to your project by importing it from Grommet and
+placing it around the “hello world” text.
+
+```js
+// App.js
+import { Grommet, Box, Heading, Text, Grid } from 'grommet';
+import { hpe } from 'grommet-theme-hpe';
+...
+return (
+  <Grommet theme={hpe}>
+    <Grid>
+      <Text>hello world</Text>
+    </Grid>
+  </Grommet>
+);
+```
+
+<br />
+
+Let's also add a heading to the page.
+
+```js
+<Grommet theme={hpe}>
+  <Heading level="1">Products</Heading>
```

Comment:
Do we have any best practices/preference about using "number" vs "string" for level? I feel like I typically use number when developing.

Reply:
changing